lib/configurators/aws/aws_test.go

```diff
@@ -655,12 +655,62 @@ func TestAWSIAMDocuments(t *testing.T) {
 				},
 			},
 		},
+		"Redshift Serverless discovery (discovery service)": {
```

Comment:
do we need to add test cases for redshift clusters, memorydb, and elasticache discovery service?

Reply:
will add